### PR TITLE
Fixed error message if a pack cannot be found in current index.pidx

### PR DESCRIPTION
**Pull request by thorstendb-ARM**
see https://github.com/Open-CMSIS-Pack/cpackget/issues/206

### CODE REVIEW

File: cmd/commands/init_test.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	{
+		name: "test create using an index.pidx",
+		args: []string{"init"},
+		setUpFunc: func(t *TestCase) {
+			server := NewServer()
+			t.args = append(t.args, server.URL()+"index.pidx")
+			server.AddRoute("index.pidx", []byte(`<?xml version="1.0" encoding="UTF-8" ?>
+<index schemaVersion="1.1.0" xs:noNamespaceSchemaLocation="PackIndex.xsd" xmlns:xs="http://www.w3.org/2001/XMLSchema-instance">
+<vendor>TheVendor</vendor>
+<url>http://the.vendor/</url>
```
Comment (jkrech) [inline cmd/commands/init_test.go:43]: should we be replacing http:// with https:// in our test files?

**Review by jkrech — APPROVED** [review]
see comment.

File: cmd/commands/init_test.go (hunk 1 above)
Comment (thorstendb-ARM) [reply cmd/commands/init_test.go:43]: Works for https://www.w3.org/2001/XMLSchema-instance, replaced the other URLs by "https"

**Review by jkrech — APPROVED** [review]
LGTM

**Pull request merged**